fennel/lib/schema/schema.py

```diff
@@ -388,6 +388,9 @@ def to_proto(self):
 
 @struct
 class Window:
+    """Window represent a time window that encapsulates event in [begin, end)
```

Comment:
Not requesting change for now but JFYI, this docstring doesn't align with official style guide: https://peps.python.org/pep-0257/ (Also read https://peps.python.org/pep-0008/ someday).